### ISSUE TITLE
2nd Leader

### ISSUE THREAD

**Issue by iScrvffy** [kolton/d2bot-with-kolbot#2947]
im starting a second leader but all the followers still join only one leaders games

/* Join game settings
	Format: "leader's profile": ["leecher 1 profile", "leecher 2 profile", ...]
	If you want everyone to join the same leader, use "leader's profile": ["all"]
	NOTE: Use PROFILE names (profile matches window title), NOT character/account names
	leader:leecher groups need to be divided by a comma
	example:
		var JoinSettings = {
			"iscrvffy": ["Lscrvffy"],
			"HMR": ["Barb"]
		};
*/

**Comment by gtoilet** [comment]
did you make a 2nd leecher file?

**Comment by iScrvffy** [comment]
> did you make a 2nd leecher file?

i made a second Mf helper which is Barb whom is supposed to follow HMR 

instead all the followers will joing iScrvffy regardless of what i enter in these lines

**Comment by gtoilet** [comment]
yes i understand you have iscrvffy as leader 1 and they use follower.dbj entrypoint
you need to make a 2nd follower.dbj entrypoint
for the other run set to follow HMR

**Comment by iScrvffy** [comment]
> yes i understand you have iscrvffy as leader 1 and they use follower.dbj entrypoint
> you need to make a 2nd follower.dbj entrypoint
> for the other run set to follow HMR

the only dbj file im seeing relating to the follower is the one the line in the original post came from?
are you saying to make a duplicate of this and only use the "Leader 1" line?

**Comment by gtoilet** [comment]
what your first post is from the follower.dbj
you need to copy that whole file and rename it follower2.dbj
edit that entrypoint for team 2 to follow HMR

**Comment by Oregon-Snake** [comment]
you may want to try what I did.

Create a new folder, same files. Run 2 bot windows. for example, one bot program is for a cbaals, another is for baal only runs. the helpers on one bot will follow the leader on that program, the leechers on the 2nd bot will follow the leader on that.

I did this to run east ladder (with a different pickit since I can make ladder only runewords on it) and one for West non ladder. so far, works flawlessly.

**Comment by iScrvffy** [comment]
> you may want to try what I did.
> 
> Create a new folder, same files. Run 2 bot windows. for example, one bot program is for a cbaals, another is for baal only runs. the helpers on one bot will follow the leader on that program, the leechers on the 2nd bot will follow the leader on that.
> 
> I did this to run east ladder (with a different pickit since I can make ladder only runewords on it) and one for West non ladder. so far, works flawlessly.

i like the idea thank you

**Comment by Oregon-Snake** [comment]
If applicable as in my case, just change the pickits as appropriate. Hope it works
